test/global-rxjs.ts

```diff
@@ -1,4 +1,5 @@
 /**
+ *
```

Comment:
@mhevery , yes, you are right, in the previous version, we patch `Observable constructor`, so I need this to make sure in `karma dist` test run, only one `rxjs` is loaded, now we don't patch `Observable constructor` any longer, we don't need this one. I will remove it.